lib/utils.py

```diff
@@ -164,4 +164,19 @@ def find_dict_with(list_of_dicts, key, value):
                 return d;
     return None
 
+def warn(message):
+    """
+    General method to warn the user to read the message before proceeding
+    and prompt a yes or no answer.
+    
+    Args:
+        message(str): The message which will be showed to the user.
+    
+    Returns:
+        returns None if the answer is yes
```

Comment:
I think a True or False response would be more fitting. Maybe rename to `get_user_confirm` so usage would look like: 
```
if get_user_confirm('my message'):
    do_dangerous_stuff()
```